app/pods/contests/admission/cast/archived/template.hbs

```diff
@@ -6,6 +6,7 @@
     {{#each admission_contests as |contest|}}
       <div class="col-12 col-md-4">
         <ContestCard
+          class="mb-5"
```

Comment:
why not just apply this class in the contest-card component?

Reply:
Get it but, that class belongs to layout and component has nothing to do with it.

Reply:
hmm makes sense !